src/main/scala/nodes/images/CenterCornerPatcher.scala

```diff
@@ -0,0 +1,48 @@
+package nodes.images
+
+import org.apache.spark.rdd.RDD
+
+import utils.{ImageUtils, Image}
+import pipelines.FunctionNode
+
+/**
+  * Extract four corner patches and the center patch of the specified size.
+  * If flips is set to true, then horizontal flips of all 5 patches is also
+  * returned
+  *
+  * @param patchSizeX size of patch along xDim
+  * @param patchSizeY size of patch along yDim
+  * @param horizontalFlips if horizontal flips of patches should also be returned
+  * @return patches of size patchSizeX x patchSizeY
+  */
+case class CenterCornerPatcher(
```

Comment:
Well we need a function node to do this `flatMap`. And I don't think the windower is appropriate here is it gets all the matches with size patchSize (we can throw out the rest of them but that seems like same amount of code with wasted work). 

I think having `ImageUtils.crop` and `ImageUtils.flipHorizontal` make this function pretty readable, but yeah this node does seem overly-specialized